resolwe_runtime_utils.py

```diff
@@ -45,8 +46,18 @@ def save_file(key, file_name, *refs):
     given.
 
     """
+    if not os.path.isfile(file_name):
+        error('Output \'{}\' missing file: \'{}\''.format(key, file_name))
```

Comment:
I think this could be worded better. What about `Output \'{}\' has a missing file: \'{}\'`?